mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -263,9 +255,18 @@ void ConstructCRef (StringBuilder sb)
 			if (DescKind == Kind.Type)
 				return;
 
+			if (ExplicitImplMember != null) {
+				sb.Append ('$');
```

Comment:
Hmm ... well this is another case where I was simply going by long-implemented and public functionality :/ So for example, you can see the cref being generated for this member:

http://docs.go-mono.com/?link=M:System.Collections.Generic.Dictionary<TKey,TValue>$System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<TKey,TValue>>.Add

So I guess the question is ... do we have `ToEcmaCref` conform to the already-implemented functionality in monodoc, or do we change the implementation of the `EcmaHelpSource.GetPublicUrl` to comform to the points you've brought up?